pkg/render/apiserver.go

```diff
@@ -1186,6 +1186,10 @@ func (c *apiServerComponent) tigeraCustomResourcesClusterRole() *rbacv1.ClusterR
 				"remoteclusterconfigurations",
 				"managedclusters",
 				"packetcaptures",
+				"policyrecommendationscopes",
+				"policyrecommendationscopes/status",
+				"policyrecommendationscope",
```

Comment:
nit: think the singular additions are just for debugging atm, just notice to fix the api issue before merging